tests/SerializerTest.elm

```diff
@@ -6,41 +6,46 @@ import Expect exposing (Expectation)
 import Test exposing (..)
 
 
+serializeExpr_ : Expr -> String
+serializeExpr_ expr =
+    serializeExpr { explicitOr = False } expr
```

Comment:
I think we should run two different test suites, one with the option enabled and the other without.